src/Utils.php

```diff
@@ -60,36 +62,48 @@ public static function wf_crm_get_fields($var = 'fields') {
 
   /**
    * Get list of states, keyed by abbreviation rather than ID.
-   * FIXME use the api for this.
    * @param null|int|string $param
    */
   public static function wf_crm_get_states($param = NULL) {
     $ret = array();
-    if (!$param || $param === 'default') {
-      $config = CRM_Core_Config::singleton();
-      if (!$param && !empty($config->provinceLimit)) {
-        $param = implode(',', $config->provinceLimit);
+    if (!$param || $param == 'default') {
+      $provinceLimit = Utils::wf_crm_get_civi_setting('provinceLimit');
+      if (!$param && $provinceLimit) {
+        $param = (array) $provinceLimit;
       }
       else {
-        $param = (int) $config->defaultContactCountry;
+        $param = [(int) Utils::wf_crm_get_civi_setting('defaultContactCountry', 1228)];
```

Comment:
Could we just do the two lines of that method here? There are a lot of helper methods already and I would like to prevent adding more. Especially since this is method is only invoked twice.